.github/workflows/main.yml

```diff
@@ -57,15 +55,25 @@ jobs:
         run: |
           pipenv check
         continue-on-error: true
-      - uses: hadolint/hadolint-action@v2.1.0
-        with:
-          dockerfile: docker/Dockerfile
       - name: Run tests
         run: |
           cd exodus
           python manage.py test
         env:
           DJANGO_SETTINGS_MODULE: exodus.settings.dev
+
+  hadolint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hadolint/hadolint-action@v2.1.0
+        with:
+          dockerfile: docker/Dockerfile
+
+  docker_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Why separate lint and build about Docker ?
Use one workflow is more efficient for this (You build on Docker only if lint is ok)
 ```suggestion
   docker_build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: hadolint/hadolint-action@v2.1.0
        with:
          dockerfile: docker/Dockerfile
```